src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -9,16 +9,69 @@ interface Props {
 }
 
 const OwnerPanel: React.SFC<Props> = ({ imageSrc, ownerName, className }) => {
+  
+  // these will be props fix throughout when passed thru 
+  const current = 4000;
+  const need = 5000;
+  //  fix ^^^ 
+
+  
   return (
     <section className={classnames(styles.container, className)}>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
       <figure className={styles.ownerContainer}>
         <img className={styles.ownerImage} src={imageSrc} alt={ownerName} />
       </figure>
+
       <h2 className={styles.ownerName}>{ownerName}</h2>
+
+      <div className={styles.progressContainer}>
+        <div className={classnames(styles.progressBar, "progress-bar")}>
+          <div className={styles.myBar} style={{width: `${(current/need)*100}%`}}> </div>
+        </div>
+        {/* pass props down here*/}
+        <div>${current} of ${need}</div>
+      </div>
+
       <div className={styles.buttonContainer}>
         <button className={classnames(styles.button, "button--filled")}>Donate</button>
         <button className={classnames(styles.button, "button--outlined")}>Buy Gift Card</button>
       </div>
+
+      <div className={styles.summaryContainer}>
```

Comment:
Not relevant to this commit, but this section is going to be hidden for MVP. When you get info from the endpoint, if this info doesn't exist, we should hide it.